src/world.rs

```diff
@@ -413,10 +413,25 @@ impl World {
     }
 
     #[inline(always)]
-    pub(crate) fn entities_meta(&self) -> &[EntityMeta] {
+    pub(crate) fn entities_meta(&self) -> &Vec<EntityMeta> {
         &self.entities.meta
     }
 
+    /// Get the id's of all pending (free) entities
+    pub fn pending(&self) -> &Vec<u32> {
```

Comment:
`&Vec<u32>` is almost never a useful type in Rust. Prefer `&[u32]`.

Reply:
Replaced by freelist.